tests/locust/suits/predict.py

```diff
@@ -0,0 +1,19 @@
+import os
+from locust import HttpUser, task, between
+
+DATSET_ENV_VAR_NAME = "LOCUST_DRUM_DATASET"
+
+
+class MyUser(HttpUser):
+    wait_time = between(0.1, 0.5)
+    host = "http://localhost:6789/"
+    # dataset = "/home/yakov.goldberg/workspace/datarobot-user-models/tests/testdata/boston_housing.csv"
```

Comment:
Do you want to leave this comment in as an example or remove it?

Reply:
already removing

Reply:
and host as well